dask/dataframe/io/tests/test_parquet.py

```diff
@@ -2888,3 +2888,108 @@ def test_parquet_pyarrow_write_empty_metadata_append(tmpdir):
         append=True,
         ignore_divisions=True,
     )
+
+
+def test_read_write_partitioned_overwrite_is_true(tmpdir):
+    # https://github.com/dask/dask/issues/6824
+
+    # Create a Dask DataFrame if size (10000, 10) with 5 partitions and write to local
+    ddf = dd.from_pandas(
+        pd.DataFrame(
+            np.random.randint(low=0, high=100, size=(10000, 10)),
```

Comment:
Can we use a smaller DataFrame for testing purposes? Maybe (100, 10) instead?

Reply:
Sure.  I resized it in a new push.